src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Fruit> fruits = new HashMap<>();
+
+    public static Fruit getFruit(String name) {
+        return fruits.get(name);
+    }
+
+    public static void addOrUpdateFruit(String name, Fruit fruit) {
+        fruits.put(name, fruit);
+    }
+
+    public static Map<String, Fruit> getAllFruits() {
+        return new HashMap<>(fruits);
```

Comment:
Returning a new HashMap with the contents of `fruits` is safe in terms of encapsulation, but be aware that this operation has a performance cost because it involves copying all entries. If the map is large, this could be significant. If you're only providing read-only access, you could consider using `Collections.unmodifiableMap(fruits)` instead, which would provide a view of the original map that cannot be modified.

Reply:
It seems to me as reduntant, but ok